tools/tpm2_create.c

```diff
@@ -361,7 +361,7 @@ bool tpm2_tool_onstart(tpm2_options **opts) {
     setbuf(stdout, NULL);
     setvbuf (stdout, NULL, _IONBF, BUFSIZ);
 
-    *opts = tpm2_options_new("H:P:K:g:G:A:I:L:u:r:c:S:E", ARRAY_LEN(topts), topts, on_option, NULL);
+    *opts = tpm2_options_new("H:P:K:g:G:A:I:L:E:u:r:c:S", ARRAY_LEN(topts), topts, on_option, NULL);
```

Comment:
But this part is not. You are doing the opposite change of what the commit says and making "E" to require an argument.

Reply:
Ah thanks, I misunderstood this part!